packages/manager/cypress/e2e/core/cloudpulse/alerts-listing-page.spec.ts

```diff
@@ -320,29 +328,57 @@ describe('Integration Tests for CloudPulse Alerts Listing Page', () => {
             .should('be.visible')
             .click();
         });
-
       ui.actionMenuItem.findByTitle(action).should('be.visible').click();
 
-      cy.wait(alias).then(({ response }) => {
+      // verify dialog title
+      cy.get('[data-qa-drawer-title]')
+        .should('be.visible')
+        .should('have.text', `${action} ${alertName} Alert?`);
+
+      cy.get('[data-qa-drawer="true"]')
```

Comment:
```suggestion
      cy.get('[data-qa-drawer]')
```

Can't this itself work?

Reply:
updated